packages/base-contract/src/subscription_manager.ts

```diff
@@ -43,26 +35,24 @@ export abstract class ContractWrapper {
             logUtils.warn(err);
         }
     }
-    constructor(web3Wrapper: Web3Wrapper, networkId: number, blockPollingIntervalMs?: number) {
+    constructor(abi: ContractAbi, web3Wrapper: Web3Wrapper) {
+        this.abi = abi;
         this._web3Wrapper = web3Wrapper;
-        this._networkId = networkId;
-        this._blockPollingIntervalMs =
-            blockPollingIntervalMs === undefined ? constants.DEFAULT_BLOCK_POLLING_INTERVAL : blockPollingIntervalMs;
         this._filters = {};
         this._filterCallbacks = {};
         this._blockAndLogStreamerIfExists = undefined;
         this._onLogAddedSubscriptionToken = undefined;
         this._onLogRemovedSubscriptionToken = undefined;
     }
-    protected _unsubscribeAll(): void {
+    public _unsubscribeAll(): void {
```

Comment:
Now that `SubscriptionManager` is a internal class and not exposed to users of the wrappers, I'm fine with making these methods public on this class, but let's remove the underscore as that is meant to denote that the method is private.